iopipe/contrib/profiler/plugin.py

```diff
@@ -53,11 +54,13 @@ def post_invoke(self, event, context):
             self.profile.disable()
 
     def pre_report(self, report):
-        pass
-
-    def post_report(self, report):
         if self.profile is not None:
             with tempfile.NamedTemporaryFile() as stats_file:
                 self.profile.dump_stats(stats_file.name)
                 signed_request = get_signed_request(report)
                 upload_profiler_report(signed_request['signedRequest'], stats_file.file)
+                if 'jwtAccess' in signed_request:
+                    self.context.iopipe.log('@iopipe/profiler.jwtAccess', signed_request['jwtAccess'])
```

Comment:
This is saving the value in custom metrics? Don't we want to add this to the schema?